cmd/agentctl/commands/log.go

```diff
@@ -133,28 +114,21 @@ A CLI tool to connect to vppagent and set vppagent logger type.
 	return cmd
 }
 
-func RunLogSet(cli *AgentCli, opts LogSetOptions) error {
-	data, err := cli.PUT("/log/"+opts.Logger+"/"+opts.Level, nil)
-	if err != nil {
-		return fmt.Errorf("HTTP PUT request failed: %v", err)
-	}
+type LogSetOptions struct {
+	Logger string
+	Level  string
+}
 
-	type response struct {
-		Logger string `json:"logger,omitempty"`
-		Level  string `json:"level,omitempty"`
-		Error  string `json:"Error,omitempty"`
-	}
-	var resp response
-	if err := json.Unmarshal(data, &resp); err != nil {
-		return err
-	}
-	logging.Debugf("response: %+v\n", resp)
+func RunLogSet(cli agentcli.Cli, opts LogSetOptions) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	if resp.Error != "" {
-		return fmt.Errorf("SERVER: %s", resp.Error)
+	err := cli.Client().LoggerSet(ctx, opts.Logger, opts.Level)
+	if err != nil {
+		return err
 	}
 
-	fmt.Fprintf(os.Stdout, "logger %s has been set to level %s\n", resp.Logger, resp.Level)
+	fmt.Fprintf(cli.Out(), "logger %s has been set to level %s\n", opts.Logger, opts.Level)
```

Comment:
```
➜ agentctl log set nosuchlogger debug
logger nosuchlogger has been set to level debug
➜
```
But there are no such logger named "nosuchlogger". Is this ok?

Reply:
It still stores the level and if such logger is created later it uses that level, but yeah it should warn at least.